### ISSUE TITLE
[Feature]: The gif images generated from the webui are very unstable

### ISSUE THREAD

**Issue by sdqq1234** [continue-revolution/sd-webui-animatediff#19]
### Expected behavior

The gif images generated from the webui are very unstable, with serious flickering, not as smooth as the images from their git
I used animediff 1.1.0, No matter what I try, the animation is unstable
Cross attention is xFormers Flash Attention
![00040-4041125323](https://github.com/continue-revolution/sd-webui-animatediff/assets/6040274/51e06fba-0159-471e-ab2e-d78502cce083)

**Comment by gitmylo** [comment]
It doesn't appear the model is being correctly used, as movements in the original repo have clear intent, while the ones in this one appear to be completely random, as if the model weights were not correctly loaded, or something like that.

**Comment by continue-revolution** [comment]
1. remove xformers should be better
2. I will re-write the random tensor generator and see if I can reproduce their result. It will probably take another 2-3 days since I have to resolve the bugs mentioned in other issues.

**Comment by jnpatrick99** [comment]
Yeah I finally ran this extension and the results are completely different that in original repo. Not using xformers.

There's no motion, only flickering. 

Also depending on the model this extension generates two completely different scenes in one gif. Try Absolute Reality from civitai and any prompt. There will be two scenes in one gif.

**Comment by antispamtutoruser123** [comment]
I'm seeing the same results.  The original animdiff looks like gif is moving smoothly.  The web plugin just looks like it's flickering through a bunch of images.

**Comment by continue-revolution** [comment]
I have been able to fix a very idiot bug in v1.2.0 and was able to (almost) produce several examples they provide. Will close this issue for now.

**Comment by jnpatrick99** [comment]
I don't think it's fixed unfortunately. I still get a lot of flicker and multiple scenes in one gif.
Motion is very subtle, unlike in original repo.
